scripts/criu-ns

```diff
@@ -203,7 +203,7 @@ def wrap_dump():
     criu_pid = os.fork()
     if criu_pid == 0:
         run_criu(sys.argv[1:])
-    return _wait_for_process_status(pid)
+    return _wait_for_process_status(criu_pid)
```

Comment:
_wait_for_process_status returns 0 if a target process has been killed, doesn't it?

Reply:
When I test it I see the following result:
```
criu-ns dump -t `pidof test` -D /tmp/test -j -v4
echo $?
5
```

Exit code `5` appears because we have `try`/`catch` around `os.wait()`, and `os.wait()` fails as follows:

```
Traceback (most recent call last):
  File "/usr/sbin/criu-ns", line 228, in <module>
    res = wrap_dump()
  File "/usr/sbin/criu-ns", line 203, in wrap_dump
    return _wait_for_process_status(pid)
  File "/usr/sbin/criu-ns", line 68, in _wait_for_process_status
    (pid, status) = os.wait()
ChildProcessError: [Errno 10] No child processes
```

Reply:
> I think we have one more issue in _wait_for_process_status

Thanks for spotting this! You are correct.